src/components/inputs/button/button.component.tsx

```diff
@@ -59,6 +59,18 @@ export interface ButtonProps {
    * The variant to use.
    */
   variant?: ButtonVariant;
+  /**
+   * The component used for the root node. Either a string to use a HTML element or a component.
+   */
+  component?: any;
+  /**
+   * The accept attribute value is an array of strings that defines the file types the file input should accept.
+   */
+  accept?: string[];
+  /**
+   * When multiple is true, the file input allows the user to select more than one file.
+   */
+  multiple?: boolean;
```

Comment:
Do you still need this in the `Button` component?

Reply:
Deleted